components/usage/pkg/db/workspace_instance_usage_test.go

```diff
@@ -16,87 +16,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestCanCreateUsageRecords(t *testing.T) {
-	teamID := uuid.New().String()
-	teamAttributionID := db.NewTeamAttributionID(teamID)
-	instanceID := uuid.New()
-	ctx := context.Background()
-
-	scenarios := []struct {
-		Name         string
-		UsageRecords []db.WorkspaceInstanceUsage
-	}{
-		{
-			Name: "One workspace instance",
-			UsageRecords: []db.WorkspaceInstanceUsage{{
-				InstanceID:    instanceID,
-				AttributionID: teamAttributionID,
-				StartedAt:     time.Now(),
-				StoppedAt:     sql.NullTime{},
-				CreditsUsed:   0,
-				GenerationID:  0,
-			}},
-		},
-	}
-
-	for _, scenario := range scenarios {
-		conn := dbtest.ConnectForTests(t)
-		t.Run(scenario.Name, func(t *testing.T) {
-			t.Cleanup(func() {
-				require.NoError(t, conn.Where(instanceID).Delete(&db.WorkspaceInstanceUsage{}).Error)
-			})
-
-			require.NoError(t, db.CreateUsageRecords(ctx, conn, scenario.UsageRecords))
-		})
-	}
-}
-
-func TestNoErrorOnCreatingDuplicateRecords(t *testing.T) {
```

Comment:
Likewise, this is also tested in the test below which ensures we can update fields